plugins/main/host-device/host-device.go

```diff
@@ -53,6 +56,9 @@ type NetConf struct {
 	RuntimeConfig struct {
 		DeviceID string `json:"deviceID,omitempty"`
 	} `json:"runtimeConfig,omitempty"`
+
+	// for internal use
+	AuxDevice string `json:"-"` // Auxiliary device name as appears on Auxiliary bus (/sys/bus/auxiliary)
```

Comment:
If this is internal, it the first word should be lower-cased. Yay golang.

Reply:
sure, will align.

noticed that `NetConf.DPDKMode` may also need to be internal as its inferred from the system[1].
but anyhow, not related to this PR.

[1] https://github.com/containernetworking/plugins/blob/d1aada912d434472ac3f709a1685cd9acfbb617a/plugins/main/host-device/host-device.go#L82